instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -9,7 +9,16 @@ group = "io.opentelemetry.instrumentation"
 val versions: Map<String, String> by project
 val springBootVersion = versions["org.springframework.boot"]
 
+configurations.all {
+  resolutionStrategy {
+    force("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:1.35.0-SNAPSHOT")
```

Comment:
this is for local testing with https://github.com/open-telemetry/opentelemetry-java/pull/6217